src/app/widgets/button/button.widget.js

```diff
@@ -116,28 +116,48 @@
         });
 
         vm.sendCommand = function () {
-            if (this.widget.command === '') this.widget.command = true;
-            if (this.widget.command_alt === '') this.widget.command_alt = false;
-            
+            // Convert boolean-like commands to the data type of item state, to avoid comparison trouble.
+            if (typeof vm.state === "boolean") {

```

Comment:
Do we really need to duplicate the whole logic here? In my eyes better code style could be to do the logic that ends in a boolean state and then only check typeof vm.state if not boolean to convert to a string value if needed.

Reply:
I agree, but I am afraid we would end up with a refactoring of almost everything. All the other widgets have similar hacks, too. 
In fact, all these type conversion issues should be bundled in a utility service or centrally managed in OHService. But that's beyond the amount time I can spend on this project in these days, and I am lacking an adequate test setup.

Reply:
I think it's okay. but I still think, if the command is not boolean, we should allow null as a possible value

Reply:
We can add that in seond PR or just do whgen someone complains :-)